src/app/zap-templates/templates/chip/encoder-src.zapt

```diff
@@ -143,7 +145,10 @@ PacketBufferHandle encode{{asCamelCased clusterName false}}Cluster{{asType name}
        .Put8(seqNum)
        .Put8(ZCL_{{asDelimitedMacro name}}_COMMAND_ID)
     {{#chip_server_cluster_command_arguments}}
-    {{#if (isString type)}}
+    {{#if (isByteString type)}}
+       .Put(static_cast<uint8_t>({{asCamelCased label}}.Length()))
```

Comment:
Why is this case safe?  It's not, as things stand.

Reply:
Added a check above for all these cast. Thanks.